include/vcpkg/platform-expression.h

```diff
@@ -25,14 +25,14 @@ namespace vcpkg::PlatformExpression
         static Expr Or(std::vector<Expr>&& exprs);
 
         // since ExprImpl is not yet defined, we need to define the ctor and dtor in the C++ file
-        Expr(); // always true
+        Expr() noexcept; // always true
```

Comment:
I disagree with this change. These aren't nothrow default constructible or nothrow movable and I see no reason to insert code to enforce termination here.

Reply:
Why is Expr not nothrow (move) constructible?

Reply:
I think I thought there was a `std::map` in here.